build.gradle

```diff
@@ -297,12 +298,23 @@ subprojects {
         apply plugin: 'com.netflix.nebula.project'
 
         if (project.name != 'micrometer-bom') {
+            apply plugin: 'biz.aQute.bnd.builder'
+
             jar {
                 manifest.attributes.put('Automatic-Module-Name', project.name.replace('-', '.'))
                 metaInf {
                     from "$rootDir/LICENSE"
                     from "$rootDir/NOTICE"
                 }
+
+                bundle {
+                    // workaround for multi-version JARs
+                    //  see https://github.com/bndtools/bnd/issues/2227
+                    bnd '''\
+-fixupmessages: '^Classes found in the wrong directory: .*'
+-exportcontents: io.micrometer.*
+'''
```

Comment:
@jonatan-ivanov thought it would be better if we make the indentation here match the rest of the build file. I think we can do that along with `String#stripIndent()` to have the same String as now. I can take care of this and merge.